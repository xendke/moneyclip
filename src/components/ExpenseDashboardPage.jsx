import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const ExpenseDashboardPage = () => (
	<div>
		{/* <header>
			<h3>Dashboard</h3>
		</header> */}
		<ExpenseListFilters />
		<ExpenseList />
	</div>
);

export default ExpenseDashboardPage;
