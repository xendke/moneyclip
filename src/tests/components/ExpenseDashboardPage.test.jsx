import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

it('renders ExpenseDashboardPage correctly', () => {
	const wrapper = shallow(<ExpenseDashboardPage />);

	expect(wrapper).toMatchSnapshot();
});
