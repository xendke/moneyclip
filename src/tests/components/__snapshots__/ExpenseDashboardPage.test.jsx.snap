// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ExpenseDashboardPage correctly 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(Component) />
</div>
`;
